demos/browser/src/global.ts

```diff
@@ -0,0 +1,4 @@
+import { Buffer } from "buffer";
+if (typeof window !== "undefined") {
+    (window as any).Buffer = Buffer
+}
```

Comment:
this seems odd, don't we already handle this in SDK `src/globals.ts`? why is it now needed again?

Reply:
This comes from pluto-encrypted. Going to check how to fix this without requiring buffer on my side and will remove the comment later. 

Very good point actually forgot to remove it :)

Reply:
Fixed and removed!